processing/download.ts

```diff
@@ -0,0 +1,64 @@
+import { basename, join } from 'path'
+import { HASHES_DIR, OSM_DOWNLOAD_DIR } from './directories.const'
+import { readPersistent, writePersistent } from './utils/persistentData'
+export async function waitForFreshData(
+  fileURL: URL,
+  maxTries: number,
+  timeout: number
+): Promise<boolean> {
```

Comment:
```suggestion
) {
```

Teste  mal ohne die explizite Type-Angabe. Die Empfehlung ist, dass TS das selbst erkennt, dann können weniger Fehler passieren